test/browser/pages/accountDetailsEditPage.js

```diff
@@ -26,16 +26,18 @@ module.exports = class PlaywrightDevPage {
 
   async checkErrorText() {
     const errorText = await this.page
-      .locator("#error-summary-title")
+      .locator("h2.govuk-error-summary__title")
       .textContent();
     return errorText.trim();
   }
 
   async editSortCode(sortCode) {
     await this.page.locator("#sortCode").fill(sortCode);
+    return sortCode;
```

Comment:
Can we remove the return logic if we're only using this to populate the sortCode value